### PR TITLE
refactor(Label): EO-1083 Forms Label refactor

### PR DESCRIPTION
**Pull request by khaziwallis**
JIRA: https://jira.rax.io/browse/EO-1083

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM

Before
-------------
screen shot 2016-08-18 at 4 27 52 pm

After
-------------
screen shot 2016-08-23 at 5 48 55 pm

### CODE REVIEW

**Comment by khaziwallis** [comment]
@CITguy @100stacks please confirm, am i on right track. if so i need to remove `Labels` doc from `Labels and Inputs`

File: src/elements/Forms/Forms.meta.json [hunk 1]
```diff
@@ -30,6 +30,10 @@
         "text area",
         "text",
         "time",
-        "validation"
+        "validation",
+        "rxFieldName",
+        "field",
+        "name",
```
Comment (CITguy) [inline src/elements/Forms/Forms.meta.json]: Keywords should be in alphabetical order. Please update.

**Comment by CITguy** [comment]
I will defer to @ElementE for feedback on copy content.

**Comment by ElementE** [comment]
Documentation write up seems correct. Marking as LGTM - but lemme know @CITguy if you see any areas pop-up of concern

**Comment by CITguy** [comment]
I feel like the hierarchy of the bullet points could use some tweaking. 

4c562c74-6637-11e6-8a9c-910a21e7c5e2

- [ ] Please unindent everything in the blue box one level.
- [ ] Links on the page pointing to `#/components/rxForm` need updated to point to correct locations (Lines [45](https://github.com/rackerlabs/encore-ui/blob/fdc96a04ce03caae4e8e7737a190d61a04c63497/src/elements/Forms/docs/forms.docs.html#L45), [83](https://github.com/rackerlabs/encore-ui/blob/fdc96a04ce03caae4e8e7737a190d61a04c63497/src/elements/Forms/docs/forms.docs.html#L83))
- [ ] [Line 108](https://github.com/rackerlabs/encore-ui/blob/fdc96a04ce03caae4e8e7737a190d61a04c63497/src/elements/Forms/docs/forms.docs.html#L108) needs updated (directive is no longer defined in `rxForm` within API)
- [ ] List item beginning on [Line 8](https://github.com/rackerlabs/encore-ui/blob/fdc96a04ce03caae4e8e7737a190d61a04c63497/src/elements/Forms/docs/forms.docs.html#L8) should be removed (rxForm API docs no longer exist)

**Comment by 100stacks** [comment]
LGTM

**Comment by CITguy** [comment]
Dev LGTM

**Pull request merged**